src/EditorFeatures/VisualBasicTest/ReplaceDocCommentTextWithTag/ReplaceDocCommentTextWithTagTests.vb

```diff
@@ -315,5 +315,39 @@ interface I
     sub WriteLine(Of TKey)(value as TKey)
 end interface")
         End Function
+
+        <WorkItem(22278, "https://github.com/dotnet/roslyn/issues/22278")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceDocCommentTextWithTag)>
+        Public Async Function TestNotApplicableKeyword() As Task
+            Await TestMissingAsync(
+"
+''' Testing keyword interf[||]ace
+class C(Of TKey)
+end class")
+        End Function
+
+        <WorkItem(22278, "https://github.com/dotnet/roslyn/issues/22278")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceDocCommentTextWithTag)>
+        Public Async Function TestInXMLAttribute() As Task
+            Await TestMissingAsync(
+"
+''' Testing keyword inside <see lang=""nu[||]ll"">
```

Comment:
❕ This should use `Nothing` instead of `null`
❕ The attribute should be named `langword` instead of `lang`